beacon-chain/core/epoch/epoch_processing_test.go

```diff
@@ -942,6 +950,19 @@ func TestCrosslinkDelta_SomeAttested(t *testing.T) {
 				0, penalties[i])
 		}
 	}
+
+	nonAttestingIndices := []uint64{12, 23}
+	for _, i := range nonAttestingIndices {
+		// Since all these validators attested, they should get the same rewards.
+		if rewards[i] != 0 {
+			t.Errorf("Wanted reward balance %d, got %d", 0, rewards[i])
+		}
+		want := uint64(50596)
```

Comment:
What is this magic number? This will break on a future config change. Any chance we can compute it from parameters?